src/block/Slab.php

```diff
@@ -38,8 +39,8 @@ class Slab extends Transparent{
 	/** @var SlabType */
 	protected $slabType;
 
-	public function __construct(BlockIdentifierFlattened $idInfo, string $name, BlockBreakInfo $breakInfo){
-		parent::__construct($idInfo, $name . " Slab", $breakInfo);
+	public function __construct(BlockIdentifierFlattened $idInfo, string $name, BlockBreakInfo $breakInfo, ?NoteInstrument $noteblockInstrument = null){
+		parent::__construct($idInfo, $name . " Slab", $breakInfo, $noteblockInstrument ?? NoteInstrument::BASS_DRUM());
```

Comment:
Don't make any assumptions about material here. `Slab` might be any material, not just stone. `BlockFactory` should always be passing the type for abstract blocks like this one. (`WoodenSlab` is OK because that's already specialized.)